src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -110,14 +110,6 @@ void NonClientIslandWindow::SetContent(winrt::Windows::UI::Xaml::UIElement conte
 void NonClientIslandWindow::SetTitlebarContent(winrt::Windows::UI::Xaml::UIElement content)
 {
     _titlebar.Content(content);
-
-    // When the size of the titlebar content changes, we want to make sure to
-    // update the size of the drag region as well.
-    const auto fwe = content.try_as<winrt::Windows::UI::Xaml::FrameworkElement>();

```

Comment:
Why was this safe to remove?

Reply:
we had 3 places where we were registering "drag bar size changed" on _three different things_; all we really need is the one on the real drag bar space.